atm.js

```diff
@@ -1,21 +1,78 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+var atm = {
+  balance: {
+    checking: 0,
+    savings: 0,
+  },
+  listen: function(){
+    $(".account").on("click", function(){
+      var action = $(event.target).val();
+      var account = $(event.target).prevAll("h2").text();
+      var amount = $(event.target).prevAll("[type=text]").val();
+      $(event.target).prevAll("[type=text]").val("");
+      if (action == "Withdraw"){this.withdraw(account, amount)}
+      else if (action == "Deposit"){this.deposit(account, amount)}
+    }.bind(this));
+  },
+  withdraw: function(account, amount){
+    if (parseFloat(amount) > this.balance[account.toLowerCase()]){
+      alert("overdraft protection coming soon!");
+      return;
+    }
+    else if (isNaN(parseFloat(amount))){
+      alert("please enter amount in digits");
+      return;
+    }
+    this.balance[account.toLowerCase()] -= parseFloat(amount);
+    $("#"+account.toLowerCase()+ " .balance")[0].textContent = ("$" + this.balance[account.toLowerCase()]);    // These are workarounds due to problems with this.updateDisplay();
+    if (this.balance[account.toLowerCase()] === 0){$("#"+account.toLowerCase()+ " .balance").addClass("zero");}
+  },
+  deposit: function(account, amount){
+    if (isNaN(parseFloat(amount))){
+      alert("please enter amount in digits");
+      return;
+    }
+    this.balance[account.toLowerCase()] += parseFloat(amount);
+    console.log(this.balance[account.toLowerCase()]);
+    $("#"+account.toLowerCase()+ " .balance")[0].textContent = ("$" + this.balance[account.toLowerCase()]);
+    if (this.balance[account.toLowerCase()] > 0){$("#"+account.toLowerCase()+ " .balance").removeClass("zero");}
+  }
+  // updateDisplay: function(){
+  //   console.log($("#checking .balance"));
+  //   var cb = this.balance.checking;
+  //   console.log("$" + this.balance.checking);
+  //   $("#checking .balance")[0].textContent = ("fucking anything!");
+  //   $("#saving, .balance").html("$" + this.balance.saving)
+  // }
 }
-
-function checkingWithdrawal(amount){
-
-}
-
-function savingsDeposit(amount){
-
-}
-
-function savingsWithdrawal(amount){
-
-}
-
-// an eventListerner for each button, each one a "click"
\ No newline at end of file
+atm.listen();
+//
+//
+// var checking = {
```

Comment:
please remove commented out code before submitting in the future. If you want to keep it for reference, you can use a git branch to store.